StyleCop.Analyzers/StyleCop.Analyzers.Test.CSharp7/StyleCop.Analyzers.Test.CSharp7.csproj

```diff
@@ -17,7 +17,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.CodeAnalysis" Version="2.6.0" />
+    <PackageReference Include="Microsoft.CodeAnalysis" Version="2.8.2" />
```

Comment:
❓  Why not version 2.10.0?

Reply:
I've been updating only as necessary when the language version changes. We could use 2.10.0, but 2.8.2 is the minimum that works with C# 7.3.